<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>826</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QMdiArea" name="mdi">
      <property name="background">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>208</red>
         <green>208</green>
         <blue>208</blue>
        </color>
       </brush>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menu_bar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>826</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="action_new"/>
    <addaction name="action_open"/>
    <addaction name="action_close"/>
    <addaction name="separator"/>
    <addaction name="action_save"/>
    <addaction name="action_save_as"/>
    <addaction name="separator"/>
    <addaction name="action_quit"/>
   </widget>
   <widget class="QMenu" name="menu_tools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="action_statistics"/>
   </widget>
   <widget class="QMenu" name="menu_about">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="action_about_picsou"/>
    <addaction name="action_about_qt"/>
    <addaction name="separator"/>
    <addaction name="action_star_me_on_github"/>
   </widget>
   <widget class="QMenu" name="menu_view">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_view"/>
   <addaction name="menu_tools"/>
   <addaction name="menu_about"/>
  </widget>
  <widget class="QToolBar" name="tool_bar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="status_bar"/>
  <widget class="QDockWidget" name="sidebar_dock">
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTreeWidget" name="tree">
       <property name="animated">
        <bool>true</bool>
       </property>
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="action_new">
   <property name="text">
    <string>New...</string>
   </property>
  </action>
  <action name="action_open">
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="action_quit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="action_close">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="action_about_picsou">
   <property name="text">
    <string>About Picsou...</string>
   </property>
  </action>
  <action name="action_about_qt">
   <property name="text">
    <string>About Qt...</string>
   </property>
  </action>
  <action name="action_star_me_on_github">
   <property name="text">
    <string>Star me on Github !</string>
   </property>
  </action>
  <action name="action_save">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="action_save_as">
   <property name="text">
    <string>Save as...</string>
   </property>
  </action>
  <action name="action_statistics">
   <property name="text">
    <string>Statistics</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
